contracts/roles.js

```diff
@@ -0,0 +1,917 @@
+/* eslint-disable max-len */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-await-in-loop */
+/* global actions, api */
+
+const ContractName = 'roles';
+const FeeMethod = ['burn', 'issuer'];
+
+actions.createSSC = async () => {
+  const tableExists = await api.db.tableExists('instances');
+  if (tableExists === false) {
+    await api.db.createTable('instances', ['id', 'lastTickTime']);
```

Comment:
I think you can remove `'id'` from `['id', 'lastTickTime']`

If you were intending to add an index for the implicit MongoDB _id field (I'm assuming id without the underscore is a typo), there's always an index for that and you don't have to add it explicitly.

Reply:
Yes, I think we should be careful here, I've seen parts using .id vs ._id. It does seem like we want to rely on the auto index _id for the instance table.